nixpkgs.json

```diff
@@ -0,0 +1,6 @@
+{
+    "owner": "NixOS",
+    "repo": "nixpkgs",
+    "rev": "1dadefec574531f021bddc85ca3c9bc81fd52899",
+    "sha256": "0csm7nycb8p990jl1mj20mg79lxpfg2xcplndcg1dbqclgw1wmqd"
+}
```

Comment:
Terribly sorry for feature creeping on this PR, but since we now have just a robust locking of all dependencies, we need to regularly update them. Please add a GitHub Action to automatically bump this revision every month.

This is already done in https://github.com/niteoweb/kai-app/blob/master/.github/workflows/nixpkgs-update.yml, but the script that is used should probably be released first, so we don't copy/paste it around every repo.

Reply:
We could use [Niv](https://github.com/nmattia/niv) to manage Nix dependencies like that. I didn't use that originally because it just adds some more tooling up top without much benefit, but recently https://github.com/knl/niv-updater-action/ was announced, which can update Niv dependencies with a GitHub action. We could use that instead of the custom script.

Reply:
I went through the code to understand how does the action work. However, it seems the action [looks for a file names `sources.json`](https://github.com/knl/niv-updater-action/blob/master/niv-updater#L62), which I think should be `nixpkgs.json` for this repo?

Reply:
Hmm, I'm definitely missing something... I'll try it anyway.

Reply:
Not in this PR though. Creating a separate issue.

Reply:
A prerequisite for that action is to use [Niv](https://github.com/nmattia/niv), which is what creates the `sources.json`. The current `nixpkgs.json` won't be used anymore